aerolyzer/image_restriction_functions.py

```diff
@@ -0,0 +1,237 @@
+                if(self.imgKey < self.acceptImgKey):
+                    return True
+
+                #Case 3: File Size byte-type is larger
+                elif(self.imgKey > self.acceptImgKey):
+                    return False
+
+                #Case 4: File Size byte-type unrecognized
+                else:
+                    return False
+
+    '''
+    Purpose:        The purpose of this function  is to determine whether or not the image is
+                    an accepted file type.
+    Inputs:         dict exifData, list acceptedFileTypes
+    Outputs:        None
+    Returns:        Boolean
+    Assumptions:    N/A
+    '''
+    def accept_file_type(self, exifData, acceptedFileTypes):
+        #check to make sure the key-value pair exists in exifData to prevent errors
+        if('File Type' not in exifData.keys()):
+            return False
+
+        if any(exifData['File Type'].upper() in self.fileType for self.fileType in acceptedFileTypes):
+            return True
+        else:
+            return False
+
+    '''
+    Purpose:        The purpose of this function is to determine whether or not the image
+                    exceeds the minimum resolution.
+    Inputs:         dict exifData, int imgWidthMin, int imgHeightMin
+    Outputs:        None
+    Returns:        Boolean
+    Assumptions:    N/A
+    '''
+    def accept_resolution(self, exifData, imgWidthMin, imgHeightMin, imgWidthMax, imgHeightMax):
+        #check to make sure the key-value pair exists in exifData to prevent errors
+        if('Exif Image Width' not in exifData.keys() or 'Exif Image Height'in exifData.keys()):
+            return False
+        
+        if (exifData['Exif Image Width'] >= imgWidthMin and exifData['Exif Image Height'] >= imgHeightMin):
+            if (exifData['Exif Image Width'] <= imgWidthMax and exifData['Exif Image Height'] <= imgHeightMax):
+                return True
+        else:
+            return False
+
+    '''
+    Purpose:        The purpose of this function is to determine whether or not the image was
+                    taken by a device with location services enabled for the camera.
+    Inputs:         dict exifData
+    Outputs:        None
+    Returns:        Boolean
+    Assumptions:    N/A
+    '''
+    def is_location_services(self, exifData):
+        if('GPS Latitude' not in exifData.keys() or 'GPS Longitude' not in exifData.keys()):
+            return False
+        
+        if (exifData['GPS Latitude'] != '' and exifData['GPS Longitude'] != ''):
+            return True
+        else:
+            return False
+
+    '''
+    Purpose:        The purpose of this function is to parse the file_size value into numbers
+                    and letters.
+    Inputs:         string file_size
+    Outputs:        None
+    Returns:        list file_parameters (digits, spaces, letters)
+    Assumptions:    N/A
+    '''
+    def parse_file_size(self, fileSize):
```

Comment:
Convention is for private functions (i.e. functions you don't want the end user to call, but will be needed by functions within) preface with "_". E.g. _parse_filesize